content/chainguard/chainguard-enforce/installation/preflight-checklist.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Preflight Checklist for Chainguard Enforce"
+linktitle: "Preflight Checklist"
+type: "article"
+date: 2023-07-07T05:56:52-07:00
+lastmod: 2023-07-07T05:56:52-07:00
+draft: false
+tags: ["Enforce", "chainctl", "Product"]
+images: []
+menu:
+  docs:
+    parent: "installation"
+toc: true
+weight: 005
+---
+
+There are a few things you need to have in place in order to install Chainguard Enforce. However, there's more than one offering of Chainguard Enforce and there are multiple different ways to install them. Because each installation method has its own set of prerequisites, it can be confusing to know exactly what you need to have in place before you install Chainguard Enforce.
+
+This document is intended to serve as a quick reference outlining the various components you need in order to install Chainguard Enforce under each currently-available method. It starts with a general list of prerequisites across the various installation methods. It then provides a brief overview of every installation method and highlights the prerequisites needed for each.
+
+Note that, in addition to the prerequisites listed here, there are [additional requirements for the Chainguard Enforce Agent](/chainguard/chainguard-enforce/reference/installation-requirements/) as well as [network requirements](/chainguard/chainguard-enforce/reference/network-requirements/) one should consider when installing Chainguard Enforce.
+
+
+## General Installation Checklist
+
+The following is a list of prerequisites required across all the different installation methods available for Chainguard Enforce. 
+
+- [x] Every installation method requires you to have access to Chainguard Enforce. Sign up using [this inquiry form](https://www.chainguard.dev/contact?utm_source=docs) if you don't already have access.
```

Comment:
What if you removed the hyphens here and just had the checkboxes?

Reply:
I didn't think this was possible but I just learned how to do inline CSS to hide the bullets on this page (and this page alone).